src/rpc/misc.cpp

```diff
@@ -597,6 +599,49 @@ static UniValue echo(const JSONRPCRequest& request)
     return request.params;
 }
 
+void AddSummary(IndexSummary&& summary, UniValue& results)
+{
+    UniValue entry(UniValue::VOBJ);
+    entry.pushKV("name", summary.name);
+    entry.pushKV("synced", summary.synced);
+    entry.pushKV("best_block_height", summary.best_block_height);
+    results.push_back(entry);
+}
+
+static UniValue listindices(const JSONRPCRequest& request)
+{
+            RPCHelpMan{"listindices",
+                "\nReturns the available indices and their status.\n",
+                {},
+                RPCResult{
+                    RPCResult::Type::ARR, "", "",
```

Comment:
Not sure it makes sense to do so here, but ISTM returning a JSON object is generally the preferred RPC result.

Reply:
The name should be unique, so it could be the key of the json object?